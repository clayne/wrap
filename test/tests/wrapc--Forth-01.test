wrapc | /dev/null | | hello_01.fs | 0
