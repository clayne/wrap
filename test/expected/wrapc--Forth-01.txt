\ Forth is an imperative stack-based computer programming language and
\ environment originally designed by Chuck Moore. Language features include
\ structured programming, reflection (the ability to examine and modify program
\ structure during execution), concatenative programming (functions are
\ composed with juxtaposition) and extensibility (the programmer can create new
\ commands).

.( "hello, world" ) CR
